packages/workers/src/PluginWorker.ts

```diff
@@ -79,42 +75,49 @@ lockdown({
       this.rpcStream = mux.createStream(STREAM_NAMES.JSON_RPC) as any;
     }
 
-    private async onCommandRequest(message: WorkerCommandRequest) {
+    private async onCommandRequest(message: any) {
       if (!message || typeof message !== 'object' || Array.isArray(message)) {
         console.error('Command stream received non-object message.');
         return;
       }
 
-      const { id, command, data } = message;
+      const { id, method, params } = message;
 
-      if (!id || typeof id !== 'string') {
+      if (!id && typeof id !== 'string' && typeof id !== 'number') {
```

Comment:
We have complete control over the IDs that come in over this stream, so I think we can mostly keep the original implementation, and just expect either a `number` or `string`.

If we're expecting a `number`, I think we can just do the following, unless we want to e.g. enforce that the numbers are integers.
```suggestion
      if (typeof id !== 'number') {
```

Reply:
This check will permit the empty string as an `id`. Since we're using `nanoid` now, I think we can revert this and go with the original condition.